### PR TITLE
FOSSC-70 python3-pyserial

### PR DESCRIPTION
**Pull request by dierberg**

### CODE REVIEW

File: analysed-packages/pyserial/version-3.5/pyserial-3.5-OSS-disclosure.txt [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+===============================================================================
+
+LICENSES 
+
+-------------------------------------------------------------------------------
+
+BSD-3-Clause
+
+Copyright (c) <year> <owner>.
```
Comment (ParianGol) [inline analysed-packages/pyserial/version-3.5/pyserial-3.5-OSS-disclosure.txt]: please remove copyright notices from license text

File: analysed-packages/pyserial/version-3.5/pyserial-3.5-SPDX2TV.spdx [hunk 2]
```diff
@@ -0,0 +1,1125 @@
...
+DocumentNamespace: http://fossology-6674c79fbf-2qzn2/repo/SPDX2TV_pyserial-3.5.tar.gz_1716285183.spdx
+DocumentName: /srv/fossology/repository/report
+SPDXID: SPDXRef-DOCUMENT
+
+##-------------------------
+## Creation Information
+##-------------------------
+
+Creator: Tool: fossology-4.4.0
+Creator: Person: Sascha Dierberg <dierberg@dresearch-fe.de>
```
Comment (ParianGol) [inline analysed-packages/pyserial/version-3.5/pyserial-3.5-SPDX2TV.spdx]: Only name should be mentioned here and not email

Comment (ParianGol) [reply analysed-packages/pyserial/version-3.5/pyserial-3.5-SPDX2TV.spdx]:
```suggestion
Creator: Person: Sascha Dierberg
```

File: analysed-packages/pyserial/version-3.5/pyserial-3.5-SPDX2TV.spdx [hunk 3]
```diff
@@ -0,0 +1,1125 @@
...
+
+##-------------------------
+## License Information
+##-------------------------
+
+
+  
+LicenseID: LicenseRef-BSD-3-Clause
+LicenseName: BSD 3-Clause "New" or "Revised" License
+ExtractedText: <text> Copyright (c) <year> <owner>.
```
Comment (ParianGol) [inline analysed-packages/pyserial/version-3.5/pyserial-3.5-SPDX2TV.spdx]: Please remove copyright notice. After that regenerate json, rdf.xml and yaml files

Comment (ParianGol) [reply analysed-packages/pyserial/version-3.5/pyserial-3.5-SPDX2TV.spdx]:
```suggestion
ExtractedText:
```

**Review by ParianGol — COMMENTED** [review]
Apart from those comments, it looks good to me :D

File: analysed-packages/pyserial/version-3.5/pyserial-3.5-OSS-disclosure.txt (hunk 1 above)
Comment (dierberg) [reply analysed-packages/pyserial/version-3.5/pyserial-3.5-OSS-disclosure.txt]: done in Fossology server

File: analysed-packages/pyserial/version-3.5/pyserial-3.5-SPDX2TV.spdx (hunk 3 above)
Comment (dierberg) [reply analysed-packages/pyserial/version-3.5/pyserial-3.5-SPDX2TV.spdx]: done in Fossology server

**Pull request merged**